Source/SwiftLintFramework/Rules/Style/FileTypesOrderRuleExamples.swift

```diff
@@ -192,6 +202,16 @@ internal struct FileTypesOrderRuleExamples {
 
         // Main Type
         struct ContentView: View {}
+        """),
+        Example("""
+        // Library Content Provider
+        ↓struct ContentView_LibraryContent: LibraryContentProvider {}
+
+        // Main Type
+        struct ContentView: View {}
+
+        // Preview Provider
+        struct ContentView_Previews: PreviewProvider {}
```

Comment:
Can we have a different name for the struct to avoid duplicates? I like examples to be compilable in real use.

Reply:
I don't quite understand what you mean, since all types in the example had different names, but I removed the PreviewProvider since it was not necessary for the example.

Reply:
This example contained `ContentView_Previews` twice previously.